resolwe_bio/processes/support_processors/bam_split.yml

```diff
@@ -56,18 +55,23 @@
         re-error "Both or no headers must be present"
       {% endif %}
 
-      BAM={{bam.bam.file}}
+      BAM={{ bam.bam.file }}
       NAME=`basename {{ bam.bam.file }} .bam`
-      ORG1={{organism}}
-      ORG2={{organism2}}
+      BUILD={{ bam.build }}
+      BUILD1=${BUILD%/*}
+      BUILD2=${BUILD##*/}
+      SPECIES1={{ bam.species }}
+      SPECIES2=`build_species_mapper.py ${BUILD2}`  # Update when new hybrid organism
```

Comment:
This can be done directly in the process by using a [jinja dict](http://jinja.pocoo.org/docs/2.10/templates/#dict) making the `build_species_mapper.py` script redundant.